package.json

```diff
@@ -21,6 +21,7 @@
     }
   ],
   "dependencies": {
+    "es6-shim": "^0.26.1",
```

Comment:
That makes a lot of sense. We should probably include this in core ultimately, but we can leave it here while this is under development.